suite/tests/client-interface/drx-scattergather.c

```diff
@@ -503,10 +503,15 @@ test_avx2_avx512_scatter_gather(void)
         return false;
 #    endif
 #    ifdef __AVX__
-    if (!test_avx2_gather(test_avx2_vpgatherdd, ref_sparse_test_buf,
-                          ref_idx32_val32_xmm_ymm_zmm, test_idx32_vec,
-                          output_xmm_ymm_zmm))
-        return false;
+    /* Run in a loop to trigger trace creation and stress things like cloning
```

Comment:
Are we sure this will always trigger trace creation? I don't want the test to be flaky.

Reply:
Failing to make a trace wouldn't make the test fail: the output would be the same.  It just might not stress the assert path.  But, yes I think this should always make a trace, unless the default threshold were raised from 50, which is not likely, and even then the test would not be flaky as in sometimes failing.

Reply:
Indeed, okay with me then. At least it won't fail I guess :)

Reply:
Once the other PR #4877 goes in, we'll have emulation labels with callbacks for rep string loops, and a lot of those will show up in traces, so I think we'll have good coverage: so I don't consider this loop critical (plus I think years from now this will still cover the trace path).